src/MAUI/Maui.Samples/Samples/GraphicsOverlay/Animate3DGraphic/Animate3DGraphic.xaml

```diff
@@ -10,6 +10,7 @@
                 BackgroundColor="{AppThemeBinding Light=#dfdfdf,
                                                   Dark=#303030}"
                 HorizontalOptions="End"
+                WidthRequest="400"
```

Comment:
This fixes a MacCatalyst specific UI issue where the border overlaps the buttons